.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,19 +13,16 @@ After:
 - [ ] **DONE**
 
 ## Documentation
-- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
-- No documentation needed: only internal and user invisible changes
+- No documentation needed: **add explanation. This can't be used if there is a GUI diff**, one can be "only internal and user invisible cnages"
```

Comment:
```suggestion
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**, one can be "only internal and user invisible changes"
```

Reply:
Typo fixed, re-review requested.